ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -197,6 +259,35 @@ export default class StatusDropdown extends React.Component {
             }
           />
         )}
+        <FileBugModal
```

Comment:
I see we have the option to file bugs, even for improvements.
This shouldn't be allowed.

Reply:
We have to pull `alert.is_regression` from the models, which I think it's not a straightforward process. Will look into it.
I would prefer to not fix the bugs they were before this improvement if it takes too much time. :)

Reply:
On second thought, we could let it as it is. It can actually be useful to always have this option at hand.